app.js

```diff
@@ -30,6 +49,15 @@ app.get('/', async (req, res) => {
   res.json({ hello: 'world' });
 });
 
+app.get('/logout', async (req, res) => {
+  console.log('Logging out user: ', req.user);
+  req.logout();
+
+  // Update the cookie with a value that's invalid, and to expire right away
+  res.cookie('connect.sid', 'deleted', { expires: new Date(1) });
```

Comment:
Why not Date(0)?

Reply:
Honestly, no reason. I think the examples i looked at had it as `Date(1)` and i didn't bother to check if zero worked.